weave/environment.py

```diff
@@ -144,11 +146,18 @@ def weave_server_url() -> str:
 
 
 def wandb_base_url() -> str:
-    pod_url = os.getenv("WANDB_PRIVATE_BASE_URL", "").rstrip("/")
-    return pod_url if pod_url != "" else wandb_frontend_base_url()
+    pod_url = os.getenv("WANDB_PRIVATE_BASE_URL")
+    if pod_url:
+        return pod_url.rstrip("/")
+
+    settings = Settings()
+    return os.environ.get("WANDB_BASE_URL", settings.base_url).rstrip("/")
```

Comment:
this divergence from `wandb_frontend_base_url` implies something is no longer consistent

Reply:
I think I understand what you are referring to, which is that `WANDB_BASE_URL` is being used to set both gorilla and the frontend link. We currently just **assume** that the frontend is being served at `app + ` the `WANDB_BASE_URL` path. Which is obviously not always the case, as in local dev or qa. Thus, there is a legitimate divergence. 

I think setting the base url from the frontend url (what we are doing currently) is a bit weird right? Shouldn't these things not necessarily rely on each other?